Flabbergasted-Flamingos/bangazon-api
# Rb product type resource

# Description
This is the pull request for the product types

## Number of Fixes
No fixes, other than please check for merge conflicts and that naming conventions look correct

## Related Ticket(s)
initial part of ticket 6

## Problem to Solve
This pull request sets up inital model, view and serializer for ProductTypes

## Proposed Changes
This will affect the following documents:
api/models.py
api/serializers.py
api/urls.py
api/views.py


## Expected Behavior
Once up and running bangazonian should be able to add product types and GET POST product types

## Steps to Test Solution
Pull down my branch:
```
git fetch --all
git checkout rb-product_type_resource
```
Make sure you are in the directory bangazon-api

If you have an existing db.sqlite3, delete it.
Also if you currently have the server running close it down so that your port will be available

Migrate:

python manage.py api
Run server:
python manage.py runserver
Go to the following url: http://127.0.0.1:8000/api/productTypes/

Add data

Go to the url: http://127.0.0.1:8000/api/productTypes/1 and you should see the first piece of data that you added. Then you can try /2, /3, etc...
